website/content/en/docs/getting-started/users.md

```diff
@@ -115,4 +115,25 @@ To deploy ClusterLink on another cluster, please repeat steps 2-3 in a console w
 ## Try it out
 
 Check out the [ClusterLink Tutorials]({{< ref "tutorials" >}}) for setting up
- multi-cluster connectivity for applications using two or more clusters.
+multi-cluster connectivity for applications using two or more clusters.
+
+## Uninstall ClusterLink
```

Comment:
should we also include instructions for uninstalling CRDs, namespaces and the operator?

Reply:
Can be done in different PR, but because it is more complicated maybe it should be done by ClusterLink CLI?

Reply:
deploy and undeploy of all should be done in the same way (e.g., manual or by CLI). It probably means new command for the CLI if we decide to take that route. I **think** I've seen mostly instructions for manual installation of CRDs but that's not a statistically significant study, just (biased?) recollection

Reply:
I'm fine with it being a separate PR (esp if you prefer the CLI way). We should prioritize the documentation of the manual step first, though.